Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqTable.Tests.ps1

```diff
@@ -63,6 +63,76 @@ Describe "Get-BqTable" {
     }
 }
 
+Describe "Set-BqTable" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+    }
+
+    It "should update trivial metadata fields via pipeline" {
+        try {
+            New-BqTable -Dataset $test_set "pipe_table" -Name "test" -Description "my table"
```

Comment:
The point of this is the server copy has different values, so when we run Set- for the first time, they're updated.

Normally yes.  A user would just put the correct values into the New- command in the first place.